starport/services/chain/chain.go

```diff
@@ -118,6 +121,13 @@ func KeyringBackend(keyringBackend chaincmd.KeyringBackend) Option {
 	}
 }
 
+// DefaultKeyringBackend specify the keyring backend to use for the chain command by default (if no config is specified)
+func DefaultKeyringBackend(keyringBackend chaincmd.KeyringBackend) Option {
```

Comment:
I feel like having both `KeyringBackend` and `DefaultKeyringBackend` is a bit too much. I think we can have a hard coded default for OS keyring for `Chain` which can be overwritten by `DefaultKeyringBackend` or the `config.yml`.

Reply:
Yeah that's fine for me. The `DefaultKeyringBackend` is only used by Network Builder for now which is always OS